app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -125,60 +153,63 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
         )
     }
 
+    private fun onSyncNow() {
+        lifecycleScope.launch {
+            requireComponents.analytics.metrics.track(Event.SyncAccountSyncNow)
+            // Trigger a sync.
+            requireComponents.backgroundServices.accountManager.syncNowAsync().await()
+            // Poll for device events.
+            accountManager.authenticatedAccount()
+                ?.deviceConstellation()
+                ?.refreshDeviceStateAsync()
+                ?.await()
+        }
+    }
+
+    private fun makeSnackbar(newValue: String): Boolean {
```

Comment:
Rename this to describe the intent of the method. You already have `checkValidName`, so maybe just use that here as well.